src/pudl_archiver/orchestrator.py

```diff
@@ -232,9 +238,6 @@ async def _apply_changes(self):
         """Actually upload and delete what we listed in self.uploads/deletes."""
         logger.info(f"To delete: {self.deletes}")
         logger.info(f"To upload: {self.uploads}")
```

Comment:
It'd be nice to pull this logging out of `_apply_changes` so we can still see "what would have happened if we weren't doing `--dry-run`"
```suggestion
```